subprojects/android-test/test-annotations/src/main/kotlin/com/avito/android/test/annotations/Flaky.kt

```diff
@@ -0,0 +1,7 @@
+package com.avito.android.test.annotations
+
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.CLASS, AnnotationTarget.FUNCTION)
+annotation class Flaky(val reason: String = NO_REASON)
```

Comment:
i want to see link to "Dealing with flaky tests" page in doc here
even if it's a stub page for now

Reply:
Docs added